### ISSUE TITLE
[bug] Need to enter connection details twice in private mode

### ISSUE THREAD

**Issue by JCGoran** [arkadiyt/zoom-redirector#3]
If I go to Chromium incognito/Firefox private mode and try to join a meeting with this extension enabled, I need to:
1) input the connection details (display name and optionally password)
2) accept Zoom's privacy policy
3) input the same connection details again

Note that if I do a page refresh instead of inputing the details at step 1), Zoom automatically redirect me to the privacy policy page, so I imagine that this is due to some browser cookie(s) being set upon first connection to the `zoom.us` domain, any way around this?

**Comment by arkadiyt** [comment]
I'll try to dig into this tonight - thanks

**Comment by arkadiyt** [comment]
@JCGoran when I join a meeting without cookies it directs me to enter my display name and then allows me to join. Are you still able to reproduce this?

**Comment by JCGoran** [comment]
Yes, on both Firefox (ESR 68) and Chromium (81), with the extension enabled and with a fresh profile (so other extensions can't interfere), once I enter the name, it redirects me to the privacy policy, and then I need to enter the name again, see the image below for what happens.
Going directly to the URL `*/wc/join/[NUMBER]` vs. the standard `*/wc/[NUMBER]/join` has the same effect.

![Image of said bug](https://user-images.githubusercontent.com/25671271/77779262-91903e00-7052-11ea-9cd2-108303702341.png)

**Comment by JCGoran** [comment]
Okay I think this has something to do with GDPR compliance, since I'm trying to connect from an EU country.
I just tried connecting to the same meeting via VPN from various countries: Canada, Mexico, US, and on none of them have I been asked to accept the privacy policy.

**Comment by arkadiyt** [comment]
Confirmed I could repro by exiting my traffic from Belgium.

Interestingly after the GDPR prompt going to `/wc/join/<id>` is what prompts for the 2nd time, but going to `/wc/<id>/join` does not prompt again.

I've pushed up the `atetelman--gdpr` branch which redirects `/wc/join/<id>` urls to `/wc/<id>/join` which fixed the issue for me. Could you try it out and let me know if it works?

**Comment by JCGoran** [comment]
It works, but only if I don't set a password for the meeting.
With a password in place, it prompts for it, then redirects to the privacy, then it goes back to the prompt.
